### DISCUSSION TITLE
长连接情况下内存缓慢增长问题

### DISCUSSION THREAD

**Discussion by hoocoo** [alibaba/higress#893] [Q&A]
@CH3CHO 你好，请教下长链接的场景，连接数保持不变的情况下（大概一万websocket长连接），只发送心跳消息。，内存一直缓慢增长，是正常现象么？ 内存用完了会不会自动回收？ 回收长连接会不会断开？ 
![image](https://github.com/alibaba/higress/assets/31276333/9db46292-3eb6-4d51-beef-548051fbfdc3)

**Comment by CH3CHO** [comment]
这方面我不是很了解诶。你可以先dump一下内存看看。

**Reply by hoocoo** [reply]
定位到原因了，感谢

**Reply by CH3CHO** [reply]
能分享一下原因吗？
